mdpow/templates/bar_opls.mdp

```diff
@@ -1,7 +1,7 @@
-; Gromacs mdp file
-; Template for MD Hydration FEP with OPLS-AA (part of POW)
+; GROMACS mdp file
+; Template for MD Hydration FEP with OPLS-AA (part of MDPOW)
 ;
-; grompp -pp -c ../../../MD_POSRES/md.gro -p ../../../top/system.top -f fep_opls.mdp -o md.tpr
+; grompp -pp -c ../MD_NPT/md.gro -p ../top/system.top -f fep.mdp -o md.tpr
```

Comment:
Same comment as above: I don't really understand from which folder this command is executed, nor how `../../../top/system.top` can become `../top/system.top` without error, but if you have checked it it's fine.